src/main/haskell/language-kore/src/Data/Kore/ASTTraversals.hs

```diff
@@ -1,17 +1,19 @@
 {-# LANGUAGE FlexibleContexts      #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE Rank2Types            #-}
-module Data.Kore.ASTTraversals ( bottomUpVisitor
+module Data.Kore.ASTTraversals {-( bottomUpVisitor
```

Comment:
I guess this is a temporary change and it will not go like this in the master branch, right?